### ISSUE TITLE
Adorn Stone Torch Support

### ISSUE THREAD

**Issue by ardissaps** [Juuxel/VanillaParts#30]
Would you support your own mod Adorn for multipart? (possibly also other mods custom non-full block?)
Thx for the great mod.

Addition : And readable name for the object?
https://imgur.com/a/syikiz0

**Comment by ardissaps** [comment]
NVM, the it is fixed by the [WiredRedstone](https://modrinth.com/mod/wiredredstone) (same dependency), I use both works perfect.

**Comment by Juuxel** [comment]
I don't see what Wired Redstone has to do with Adorn? Adding Adorn support *is* something that I could do in the future so let's keep this open.

**Comment by ardissaps** [comment]
> I don't see what Wired Redstone has to do with Adorn? Adding Adorn support _is_ something that I could do in the future so let's keep this open.

Stone torch

**Comment by Juuxel** [comment]
Ah, I see. But I think I'll still add support for *Adorn's* stone torch at some point 🙂

**Comment by ardissaps** [comment]
Yeah! Thanks! (1.18.2), I mean Stone Torch x VanillaParts, I said wired redstone fix that for the human readable name for combined object (it was machine language, just edit the lang files)
